tools/dreport.d/include.d/functions

```diff
@@ -173,3 +173,91 @@ function fetch_serial_number() {
     fi
 }
 
+#Dump originator params
+declare -x ORIGINATOR_TYPE=""
+declare -x ORIGINATOR_ID=""
+
+#Function to get the Originator details
+#for Originator Type and Originator ID
+function get_originator_details() {
+    # A hash map for each possible dump types
+    # Add/Remove as per the requirement
+    local -A dump_name_map
+    dump_name_map["dumps"]="bmc"
+    dump_name_map["faultlogs"]="faultlog"
+    dump_name_map["hardwaredump"]="hardware"
+    dump_name_map["hostbootdump"]="hostboot"
+    dump_name_map["msbedump"]="msbe"
+    dump_name_map["sbedump"]="sbe"
+
+    dump_type_received=$(echo "$dump_dir" | cut -d'/' -f 5)
+    dump_entry_id=$(echo "$dump_dir" | cut -d'/' -f 6)
+
+    dump_type_mapped="$dump_type_received"
+
+    for key in "${!dump_name_map[@]}"
+    do
+        if [ "$key" = "$dump_type_received" ]; then
+            dump_type_mapped="${dump_name_map[$key]}"
+            break
+        fi
+    done
+
+    # If the dump type received is not in the known list/map
```

Comment:
we can simplify this like 

dump_type_mapped=""
if [ "$dump_type_mapped" = "" ]; then
        return
fi

Reply:
Addressed as suggested